mapusaurus/geo/urls.py

```diff
@@ -2,7 +2,5 @@
 
 urlpatterns = patterns(
     '',
-    url(r'^tractCentroids/(?P<northEastLat>-?\d+\.\d{6})/(?P<northEastLon>-?\d+\.\d{6})/(?P<southWestLat>-?\d+\.\d{6})/(?P<southWestLon>-?\d+\.\d{6})$', 
-        'geo.views.tract_centroids_as_json', name='tract_centroids'),
```

Comment:
Do you not need a url for the function that replaced this, `format_geo_to_geojson`?